libs/designsystem/src/lib/components/list/list.component.html

```diff
@@ -79,7 +79,12 @@
 
 <ng-template #virtualItemsTemplate let-virtualScrollData>
   <div *uiScroll="let item of virtualScrollData; let i = index">
-    <div [class.item-wrapper]="!item.headingName" [class.section-header]="item.headingName">
+    <div
+      [class.item-wrapper]="!item.headingName"
+      [class.section-header]="!!item.headingName"
+      [class.virtual-section-item]="_isSectionsEnabled && !item.headingName"
+      [ngClass]="_getBoundaryClass(i)"
+    >
```

Comment:
Why does this wrapper exist? Couldn't these classes just as well be added to the virtual scroll container?